README.md

```diff
@@ -39,6 +39,17 @@ In the training phase, the best case is that we are able to detect between Y+D a
 
 The labeling technical details are described [in this issue](https://github.com/marco-c/autowebcompat/issues/2).
 
+The Bounded-Box labeling allows us to store the areas where the incompatibilities lie.
+
+<img src="https://user-images.githubusercontent.com/18056781/39081659-fdd4655e-4562-11e8-86f9-a5fab28634bf.JPG" />
+
+<img src="https://user-images.githubusercontent.com/18056781/39081665-10eda006-4563-11e8-9455-986b5a23934e.jpg" />
+
+- Press 'y' to mark compatible.
+- Press 'Enter' to select regions of incompatibility.
+- Click 'T' on top left of boundary boxes to toggle between colors. GREEN --> 'n' YELLOW --> 'd'
```

Comment:
on the top left corner of the bounding-boxes ... colors. Green corresponds to 'n', yellow corresponds to 'd'.

(It would also be nice to make `Green` actually green and `yellow` actually yellow)

Reply:
@marco-c github doesn't allow colored text in markdown. It's an ongoing issue and as far as i have seen there is no way around it .